bindings/nodejs/types/account.ts

```diff
@@ -15,8 +15,14 @@ export interface AccountBalance {
 }
 
 export interface AccountSyncOptions {
-    addressIndex?: number;
-    gapLimit?: number;
+    addresses: string[]
+    addressStartIndex: number;
+    automaticOutputConsolidation: boolean;
+    forceSyncing: boolean;
+    syncPendingTransactions: boolean;
+    syncAliasesAndNfts: boolean;
+    tryCollectOutputs: boolean;
+    outputConsolidationThreshold: number;
```

Comment:
I think they are all optional

Reply:
They don't seem optional here: https://thoralf-m.github.io/wallet.rs/iota_wallet/account/operations/syncing/options/struct.SyncOptions.html

Are they optional because `serde` can provide a default value for them?

Reply:
Yes, serde will set the default values during deserialization if no other value is provided